bundles/org.corpus_tools.hexatomic.graph/src/main/java/org/corpus_tools/hexatomic/graph/GraphEditor.java

```diff
@@ -908,13 +932,15 @@ public void modifyText(ModifyEvent e) {
 
     @Override
     public void progressStarted(ProgressEvent e) {
-      // We only start to scroll after the task has ended, but we still have to implement all
+      // We only start to scroll after the task has ended, but we still have to
+      // implement all
       // methods of the interface.
     }
 
     @Override
     public void progressUpdated(ProgressEvent e) {
-      // We only start to scroll after the task has ended, but we still have to implement all
+      // We only start to scroll after the task has ended, but we still have to
+      // implement all
       // methods of the interface.
```

Comment:
```suggestion
      // implement all methods of the interface.
```

Minor formatting fix